contracts/test/unit/oracle/VoltSystemOracle.t.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {Vm} from "./../utils/Vm.sol";
+import {DSTest} from "./../utils/DSTest.sol";
+import {getCore, getAddresses, FeiTestAddresses} from "./../utils/Fixtures.sol";
+import {MockScalingPriceOracle} from "../../../mock/MockScalingPriceOracle.sol";
+import {VoltSystemOracle} from "../../../oracle/VoltSystemOracle.sol";
+import {ScalingPriceOracle} from "../../../oracle/ScalingPriceOracle.sol";
+import {MockChainlinkToken} from "../../../mock/MockChainlinkToken.sol";
+import {Decimal} from "./../../../external/Decimal.sol";
+import {Constants} from "./../../../Constants.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {LinkTokenInterface} from "@chainlink/contracts/src/v0.8/interfaces/LinkTokenInterface.sol";
+
+contract VoltSystemOracleTest is DSTest {
+    using Decimal for Decimal.D256;
+    using SafeCast for *;
+
+    /// @notice reference to the volt system oracle
+    VoltSystemOracle private voltSystemOracle;
+
+    /// @notice increase price by 3.09% per month
+    int256 public constant monthlyChangeRateBasisPoints = 309;
```

Comment:
We're not using this anywhere are we, otherwise this confuses me a bit, because isn't it just meant to set the rate once and then interpolate linearly over the year?